BDR_PATTERN.md

```
@@ -0,0 +1,861 @@
+  // Dynamic business rules through injected service
+  public function getBusinessPriority(): string
+  {
+  // Impossible with ORM entities - depends on external service
+  // Test environment: Relaxed thresholds (e.g., high priority at $100+)
+  // Production: Strict thresholds (e.g., high priority at $10,000+)
+  // Peak season: Different thresholds
+  // VIP customers: Special rules apply
+  return $this->ruleEngine->calculatePriority($this);
+  }
+  }
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)